### DISCUSSION TITLE
iOS rotate option or ability to enlarge viewing area by using two fingers

### DISCUSSION THREAD

**Discussion by evanscybersecurity** [standardnotes/forum#3328] [Suggestions] [answered]
When using iOS version of Standard Notes it is not possible to view screen contents in rotated view when enabled.

Please update code to allow for a rotational view or ability to enlarge viewing area by using two fingers

Recently I was viewing a note that had screenshots that were imported. They were really hard to see and I had no way to increase viewing area since rotation and enlarging with two fingers does not work.

I searched the best I could through the current open items for iOS. Sorry If I missed this topic from an already submitted post.

**Comment by effieeee** [comment, marked as answer]
Hi @uYtLD9zS, I'm afraid there's no way to zoom in at the moment, and rotating to landscape mode is still disabled. However, I've converted this into a **suggestion** in particular, to keep it as a feature request of sorts 🙏
